src/test/fixtures/upgrades/processed-3.json

```diff
@@ -71,6 +71,48 @@
       }
     ]
   },
+  "counters": [
```

Comment:
basically same reasoning here: this profile is version 4 and they were added in processed format version 19.

Reply:
ah yeah, oops :)